test/relay.test.js

```diff
@@ -0,0 +1,236 @@
+
+    taskConnection = connection.connectionType;
+
+    userType = new GraphQLObjectType({
+      name: 'User',
+      fields: {
+        id: globalIdField('User'),
+        name: {
+          type: GraphQLString
+        },
+        tasks: {
+          type: taskConnection,
+          args: connectionArgs,
+          resolve: resolver(User.Tasks)
+        }
+      },
+      interfaces: [nodeInterface]
+    });
+
+    schema = new GraphQLSchema({
+      query: new GraphQLObjectType({
+        name: 'RootQueryType',
+        fields: {
+          user: {
+            type: userType,
+            args: {
+              id: {
+                type: new GraphQLNonNull(GraphQLInt)
+              }
+            },
+            resolve: resolver(User)
+          },
+          users: {
+            type: new GraphQLList(userType),
+            args: {
+              limit: {
+                type: GraphQLInt
+              },
+              order: {
+                type: GraphQLString
+              }
+            },
+            resolve: resolver(User)
+          }
+        }
+      })
+    });
+
+  });
+
+  before(function () {
+    var userId = 1
+      , taskId = 1;
+
+    return this.sequelize.sync({force: true}).bind(this).then(function () {
+      return Promise.join(
+        User.create({
+          id: userId++,
+          name: 'b' + Math.random().toString(),
+          tasks: [generateTask(taskId++), generateTask(taskId++), generateTask(taskId++)]
+        }, {
+          include: [User.Tasks]
+        }),
+        User.create({
+          id: userId++,
+          name: 'a' + Math.random().toString(),
+          tasks: [generateTask(taskId++), generateTask(taskId++)]
+        }, {
+          include: [User.Tasks]
+        })
+      ).bind(this).spread(function (userA, userB) {
+          this.userA = userA;
+          this.userB = userB;
+          this.users = [userA, userB];
+        });
+    });
+  });
+
+  it('should resolve a plain result with a single model', function () {
+    var user = this.userB;
+
+    return graphql(schema, `
+      {
+        user(id: ${user.id}) {
+          name
+          tasks {
+            edges {
+              node {
+                name
+              }
+            }
+          }
+        }
+      }
+    `).then(function (result) {
+      if (result.errors) throw new Error(result.errors[0].stack);
+
+      expect(result.data).to.deep.equal({
+        user: {
+          name: user.name,
+          tasks: {
+            edges: [
+              {
+                node: {
+                  name: user.tasks[0].name
+                }
+              },
+              {
+                node: {
+                  name: user.tasks[1].name
+                }
+              }
+            ]
+          }
+        }
+      });
+    });
+  });
+
+  it('should resolve a plain result with a single model', function () {
+    var users = this.users;
+
+    return graphql(schema, `
+      {
+        users {
```

Comment:
Had an issue where nested connections were being added to attributes inside of include generator. Fixed this by checking for the existence of an edges field on each field passed in the simpleAST.